src/Tribe/Organizer.php

```diff
@@ -640,7 +640,9 @@ public static function get_fetch_callback( $event ) {
 		}
 
 		return static function () use ( $event ) {
-			$organizer_ids = array_map( 'absint', (array) get_post_meta( $event, '_EventOrganizerID' ) );
+			$organizer_ids = array_filter(
+				array_map( 'absint', (array) get_post_meta( $event, '_EventOrganizerID' ) )
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: A cast statement must not be followed by a space
* :warning: Line exceeds 80 characters; contains 91 characters

<sub>hash: 32be4997d5277f86616045e54f4a56d7</sub>